src/sentry/migrations/0582_api_tokens_aadd_name_and_last_four.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.20 on 2023-10-24 23:47
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0581_add_user_and_team_to_alert_rules"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="apitoken",
+            name="name",
+            field=models.CharField(max_length=255, null=True),
+        ),
+        migrations.AddField(
+            model_name="apitoken",
+            name="token_last_characters",
+            field=models.CharField(max_length=4, null=True),
+        ),
```

Comment:
Do you need these fields for authentication flows? `ApiToken` is replicated to relevant regions into the `ApiTokenReplica` model. If you need these fields in login flows we'll need to also update that model's schema and trigger a replication backfill.

Reply:
No, these fields won't be used in authenticating the token. Just when creating/displaying the token in the UI.